### ISSUE TITLE
Missing Visual Focus on Anchor Links

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#4738]
The top links could have the outline, but I can't see it, it has a black background. 

But the "Home" does not have the outline. 

![Screen Shot 2019-09-25 at 10 00 25 AM](https://user-images.githubusercontent.com/45409515/65608008-819c8200-df7b-11e9-828c-63be29312298.png)

**Comment by KateChuen** [comment]
OK, I see the outline using the keyboard.
